addons/website_slides/wizard/slide_channel_invite.py

```diff
@@ -86,8 +86,12 @@ def action_invite(self):
 
     def _prepare_mail_values(self, slide_channel_partner):
         """ Create mail specific for recipient """
-        subject = self._render_field('subject', slide_channel_partner.ids)[slide_channel_partner.id]
-        body = self._render_field('body', slide_channel_partner.ids)[slide_channel_partner.id]
+        set_lang, lang_opt = {}, {}
+        if (self.body_has_template_value):
```

Comment:
The `body_has_template_value` check is necessary because if the subject is unchanged, but the body is, the subject will still be translated (but not the body).

Actually, it probably break the other way around (changed subject, unchanged body). I assumed `body_has_template_value` would also check the subject but I guess not. I will check.

Reply:
ah ok, makes sense then if we want in this case to have everything in same language